privacyidea/lib/policy.py

```diff
@@ -615,6 +623,11 @@ def get_policies(self, name=None, scope=None, realm=None, active=None,
             log.debug("Policies after matching client".format(
                 reduced_policies))
 
+        if user_object is not None:
```

Comment:
Bascially we should **always** call ``filter_policy_by_conditions`` if we are **matching** policies for a behaviour decision.
We should **not** call this, if we list policies for administrative purposes.

Thanks! We should differentiate between a method ``list_policies`` and ``match_policies``.

Reply:
done in af2e33e522feb0a8c9aaee3db90f4a48aed55c26